server/src/test/java/org/opensearch/indices/replication/MergedSegmentReplicationTargetTests.java

```diff
@@ -190,7 +213,7 @@ public void onFailure(Exception e) {
         }, mock(BiConsumer.class));
     }
 
-    public void testFailure_differentSegmentFiles() throws IOException {
+    private void testFailure_differentSegmentFiles(MergedSegmentCheckpoint checkpointMergedSegment) throws IOException {
```

Comment:
testFailure_differentSegmentFiles is an acceptable naming pattern for tests, but if you are making a private it would be better to use the camel case naming.